app/models/document.rb

```diff
@@ -1,27 +1,14 @@
-class AbstractDocument
+class Document
   attr_reader :title
 
-  def self.date_metadata_keys
-    []
-  end
-
-  def self.tag_metadata_keys
-    []
-  end
-
-  def self.metadata_keys
-    tag_metadata_keys + date_metadata_keys
-  end
-
-  def self.metadata_name_mappings
-    {}
-  end
-
-  def initialize(attrs)
+  def initialize(attrs, finder)
+    attrs = attrs.with_indifferent_access
```

Comment:
For reference for anyone else this was discussed offline. This line was originally in https://github.com/alphagov/finder-frontend/blob/master/app/parsers/document_parser.rb#L3 but because we don't have a `DocumentParser` anymore I moved this line to the initialize method of `Document`.